commands/slash/reload.js

```diff
@@ -77,7 +77,7 @@ const command = new SlashCommand()
         embeds: [
           new MessageEmbed()
             .setColor(client.config.embedColor)
-            .setDescription("You are not authorized to use this command!"),
+            .setTitle("**You are not authorized to use this command!**"),
```

Comment:
The embed for setTitle is a little bit to big on mobile devices. changing this to the original setDescription instead would be better.

Reply:
Alright will fix it